docs/api/datasets.rst

```diff
@@ -127,6 +127,11 @@ So2Sat
 
 .. autoclass:: So2Sat
 
+Spacenet 1: Building Detection v1
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
SpaceNet
^^^^^^^^
```
At some point, we'll add more SpaceNet datasets and put them under this same header like we do with Landsat.